eng/tools/versioning/VersionUtils.js

```diff
@@ -38,14 +38,21 @@ function updateChangelog(
   repoRoot,
   newVersion,
   unreleased,
-  replaceVersion,
+  replaceLatestVersionTitle,
   releaseDate = null
 ) {
   const service = path.basename(path.dirname(targetPackagePath));
   const updateChangelogPath = path.resolve(
     path.join(repoRoot, "eng/common/scripts/Update-ChangeLog.ps1")
   );
-  let args = [updateChangelogPath, newVersion, service, packageName, unreleased, replaceVersion];
+  let args = [
+    updateChangelogPath,
+    newVersion,
+    service,
+    packageName,
+    unreleased,
+    replaceLatestVersionTitle
+  ];
```

Comment:
```suggestion
  let args = [
   "--ChangeLogPath",
    updateChangelogPath,
    "--Version", 
    newVersion,
    "--ServiceDirectory",
    service,
    "--PackageName",
    packageName,
    "--Unreleased",
    unreleased,
    "--ReplaceLatestEntryTitle"
    replaceLatestVersionTitle
  ];
```

Reply:
This also does not work for the boolean values. Because you cannot correctly include the powershell '$'

Reply:
Again I was not able to do this because `updateChangelogPath` is not an argument but rather the location of the script to run. And since we cant pass that with a parameter names the others also don't work with the parameter names.

Reply:
I incorrectly passed `ChangeLogPath` in my original suggestion but once I removed that everything appeared to work in my local testing. Can you please try this:
```
  let args = [
    updateChangelogPath,
    "--Version", 
    newVersion,
    "--ServiceDirectory",
    service,
    "--PackageName",
    packageName,
    "--Unreleased",
    unreleased,
    "--ReplaceLatestEntryTitle",
    replaceLatestVersionTitle
  ];
```

Reply:
I tried it again. It does not work with the boolean parameters in Update-ChangeLog.ps1. The problem seems to be that you cannot send PowerShell style boolean from js.

Reply:
looks like spawnsync is what invalidates the boolean

Reply:
Weird I just did some playing around with this and you are correct about the boolean value cannot be passed this way and I think using the `:` might be the only way. This is apparently a bug in Powershell. I found some information about it at https://stackoverflow.com/questions/5079413/how-to-pass-boolean-values-to-a-powershell-script-from-a-command-prompt. 

In any case you can still keep the name of the parameters here but you will have to use the `:` to pass the boolean values.

Reply:
I added the names for the other parameters. It doesn't work if I add parameters for the Boolean parameters.

Reply:
For the boolean you will have to do `"--ReplaceLatestEntryTitle:" + replaceLatestVersionTitle`